gcloud/datastore/query.py

```diff
@@ -29,13 +29,13 @@ class Query(object):
     This class serves as an abstraction for creating a query over data
     stored in the Cloud Datastore.
 
-    :type kind: string
-    :param kind: The kind to query.
-
     :type dataset_id: string
     :param dataset_id: The ID of the dataset to query.  If not passed,
                        uses the implicit default.
 
+    :type kind: string
+    :param kind: The kind to query.
+
```

Comment:
D'oh! I blindly changed it because I thought it needed to move. Turns out it was wrong before.